src/pages/markets/instruments/_minor-pairs-options.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import Symbol from '../components/helper/_symbol'
+import { minor_pairs_options } from './_market-symbols'
+
+const MinorPairsOptions = () => {
+    return (
+        <>
+            {minor_pairs_options.map((symbol, index) => (
+                <Symbol key={index} src={symbol.src} text={symbol.text} />
+            ))}
+        </>
+    )
+}
+
+export default MinorPairsOptions
```

Comment:
Can we have just one `_symbol.js` component that takes in a market symbol and renders all these? We can remove a lot of files if we refactor it.

Reply:
Yeah that would be nice. Should we do it in this PR or another card?

Reply:
Added a TODO for it